signer/tests/integration/block_notifier.rs

```diff
@@ -0,0 +1,72 @@
+use futures::StreamExt;
+use signer::block_notifier::{BlockNotifier, ElectrumBlockNotifier};
+use signer::config::BlockNotifierConfig;
+use tracing::{error, info, warn, Level};
+use tracing_subscriber;
+
+// TODO: need to fix docker-compose.test.yml setup so bitcoin and bitcoind
```

Comment:
Isn't the `bitcoin` and `bitcoind` essentially the same container? Otherwise, it would be great if you could create a ticket for this and reference it in the TODO comment.